src/itype.h

```diff
@@ -409,7 +409,7 @@ struct islot_gun : common_ranged_data {
     int reload_noise_volume = 0;
 
     /** Maximum aim achievable using base weapon sights */
-    int sight_dispersion = 120;
+    int sight_dispersion = 75;
```

Comment:
Why is this hidden limiter here?  Cant we  just use the actual gun sight dispersion, this just seem unfriendly to modding.

Why do we assume that all factory-issue iron sights must be bad?

Reply:
Good question. I think that it simulates inability to player precisely see distant targets in sight. It make sense.

Reply:
I'll lower it.

Reply:
Then why not add the penalty if the actual shot distance supprasses a given treshold. (Maybe taking player PER into account)

That way, pistols and shotguns will not suffer from this.

Reply:
I propose to leave it for next PR. Lowering it 35 helped already.